src/core/main.c

```diff
@@ -1364,7 +1363,6 @@ int main(int argc, char *argv[]) {
 
                 /* Running outside of a container as PID 1 */
                 arg_system = true;
-                make_null_stdio();
```

Comment:
BTW, with this patch make_null_stdio() is called even if systemd is run inside a container.

I think it makes sense since systemd is not supposed to use stdout, stderr directly.